README.md

```diff
@@ -83,19 +83,21 @@ This is an open source project managed by the Algorand Foundation. See the [cont
 
 ## Prerequisites
 
-The key required dependency is Python 3.10+, but some of the installation options below will install that for you.
-
-AlgoKit also has some runtime dependencies that also need to be available for particular commands.
+AlgoKit has some runtime dependencies that also need to be available for particular commands.
 
 > **Note**
-> You can still install and use AlgoKit without these dependencies and AlgoKit will tell you if you are missing one for a given command.
+> You can still install and use AlgoKit without these dependencies, and AlgoKit will tell you if you are missing one for a given command.
 
 - Git - Git is used when creating and updating projects from templates
 - Docker - Docker Compose (and by association, Docker) is used to run the AlgoKit LocalNet environment, we require Docker Compose 2.5.0+
 
+> **Note**
+> If you are using pipx to install AlgoKit, the key required dependency is Python 3.10+.
+
+
 ## Cross-platform installation
 
-AlgoKit can be installed using OS specific package managers, or using the python tool [pipx](https://pypa.github.io/pipx/) see below for specific installation instructions.
+AlgoKit can be installed using OS specific package managers, or using the python tool [pipx](https://pypa.github.io/pipx/). See below for specific installation instructions.
```

Comment:
Should the installation methods document the processor architectures we support?
One thing I did notice for windows was the binary built on the GitHub agent, seemed to work fine on my Windows ARM VM.

Reply:
the processor architectures we support are now documented. I stick to not supporting Windows ARM for now. If we think there is no issue supporting that then maybe we also need to change its release artifact name as well.